pkg/cloudprovider/controller/nodecontroller_test.go

```diff
@@ -704,6 +729,12 @@ func TestSyncNodeStatusTransitionTime(t *testing.T) {
 									Reason:             "Node health check succeeded: kubelet /healthz endpoint returns ok",
 									LastTransitionTime: util.Date(2012, 1, 1, 0, 0, 0, 0, time.UTC),
 								},
+								{
+									Type:               api.NodeSchedulable,
+									Status:             api.ConditionFull,
```

Comment:
The use case we are testing here is to verify last transition time change for the node condition. For this scenario, test case setup:
(a) Current node status is schedulable 
(b) Now user desired node to be not schedulable (updated in spec) but status is not updated yet  to reflect this desire.
Test execution:
   Test case will update the status (obeys the spec and marks NodeSchedulable to ConditionNone) and ensures there is change in last transition time.